MAINTAINERS

```diff
@@ -164,6 +164,7 @@ on disputes for technical matters."
 			"justincormack",
 			"riyazdf",
 			"rn",
+			"TiejunChina",
```

Comment:
This is the MAINTAINERS file, not the the `.mailmap` file as I suggested you to modify.

Can this also be a separate commit, not lumped in with unrelated changes?

Reply:
If your preferred e-mail address is `tiejun.china@gmail.com`, you can add this line to the `.mailmap`:

```
Tiejun Chen <tiejun.china@gmail.com> <tiejunc@vmware.com>
```

Patch:

```patch
diff --git a/.mailmap b/.mailmap
index 588c67bc6..1a666bd07 100644
--- a/.mailmap
+++ b/.mailmap
@@ -49,5 +49,6 @@ Sebastiaan van Stijn <sebastiaan.vanstijn@docker.com> <github@gone.nl>
 Simon Ferquel <simon.ferquel@docker.com> <simon.ferquel@hotmail.fr>
 Thomas Gazagnaire <thomas.gazagnaire@docker.com> <thomas@gazagnaire.com>
 Thomas Gazagnaire <thomas.gazagnaire@docker.com> <thomas@gazagnaire.org>
+Tiejun Chen <tiejun.china@gmail.com> <tiejunc@vmware.com>
 Vincent Demeester <Vincent.Demeester@docker.com> <vincent@sbr.pm>
 Vincent Demeester <Vincent.Demeester@docker.com> <vdemeester@docker.com>
```

That will updated the existing entry in the `AUTHORS` file when it's re-generated. You can re-generate that file using `scripts/generate-authors.sh`

Reply:
@rn @thaJeztah Okay.